src/main/kotlin/uk/gov/justice/digital/assessments/api/AssessmentAnswersDto.kt

```diff
@@ -0,0 +1,10 @@
+package uk.gov.justice.digital.assessments.api
+
+import java.util.*
+
+data class AssessmentAnswersDto (
+
+        val assessmentUuid: UUID,
+        //Question Code -> List Of Answer Codes
+        val answers: Map<String, Set<String>>
+        )
```

Comment:
I wonder if this should look at little more like the shape AssessmentEpisodeDto? and the AnswerDtos it contains. Perhaps even use the AnswerDto?  

Not important at this stage though.